test/apps/iodemo/ucx_wrapper.h

```diff
@@ -308,6 +310,16 @@ class UcxConnection {
         return _conn_id;
     }
 
+    uint64_t remote_id() const {
+        return _remote_conn_id;
+    }
+
+    void set_remote_id(uint64_t remote_conn_id) {
+        assert(_remote_conn_id == std::numeric_limits<uint64_t>::max());
```

Comment:
maybe set `_remote_conn_id` unconditionally and assert prev value equal to `max()` or new value?

Reply:
done